src/main/webapp/js/components/interface/drawer/TabbedDrawer.js

```diff
@@ -79,90 +78,117 @@ define(function (require) {
                 );
             }, this);
             return renderedLabels;
-        }
+        };
 
         // function to render all the childs, wrap each one of them in a div and manage with display
         // which child has to be visible.
-        renderMyChildren() {
-            var renderedComponents = this.props.children.map(function(child, index) {
-                if(this.state.drawerOpened == true) {
-                    var ElementToRender = child;
-                    var MyElement = (this.state.selectedTab != index) ? <div className="hiddenComponent" key={index}><ElementToRender /></div> : <div key={index}><ElementToRender /></div>;
-
-                } else {
-                    var ElementToRender = child;
-                    var MyElement = <div key={index}><ElementToRender className="hiddenComponent" /></div>;
-                }
-                return (MyElement);
-            }, this);
-            return renderedComponents;
-        }
+        renderMyChildren = () => {
+            var paddingChildren = 45;
+            if (this.props.children != undefined) {
+                var renderedComponents = this.props.children.map(function (child, index) {
+                    var ElementToRender = undefined;
+                    var MyElement = undefined;
```

Comment:
I know this variable was already named 'MyElement' but maybe we can choose a more descriptive name...

Reply:
Btw, probably you are aware but javascript has no block scope only function scope. I mean there is no need to declare the variables outside the `if` to make it available for the whole function. If you did it on purpose for the sake of clarity of something like that I am fine with that

Reply:
Names changed to:
ElementToRender -> DrawerChild
MyElement -> ComponentToRender

On the second note, it was for my peace of mind to keep things in order.

Reply:
Perfect!